src/frontc/cabs2cil.ml

```diff
@@ -3681,7 +3677,9 @@ and doExp (asconst: bool)   (* This expression is used as a constant *)
             (* Maybe it ends in U or UL. Strip those *)
             let l = String.length str in
             let baseint, kind =
-              if hasSuffix str "L" then
+              if hasSuffix str "F128" then
```

Comment:
Where is this `F128` taken from? GCC docs linked above say `q` or `Q` from what I can see.

Reply:
At the very top of the link you found, it gives the suffixes for the `_FloatN` type. I'll have to check if the preprocessor turns `q` into this as well.